### PR TITLE
chore: Fixes some tests to work cross platform

### PR DESCRIPTION
**Pull request by jmyrland**
On windows using `node@16.15.0`, 3 tests are failing.

Adjusts tests slightly to make them pass in this environment.

### CODE REVIEW

File: tests/persist.test.js [hunk 1]
```diff
@@ -1114,51 +1114,51 @@ test("multiple changes don't cause concurrent persist operations", async () => {
...
+  await wait(16);
 
   expect(memoryStorage.store['[EasyPeasyStore][0]']).toBeUndefined();
 
   store.getActions().change({
     counter: 5,
   });
 
-  await wait(20);
+  await wait(17);
```
Comment (jmyrland) [inline tests/persist.test.js:1149]: Not ideal, but it works on my machine 🤷

**Comment by jmyrland** [comment]
[I broke it](https://github.com/ctrlplusb/easy-peasy/actions/runs/3096631134/jobs/5012402806#step:4:62) 😢

File: tests/persist.test.js (hunk 1 above)
Comment (jmyrland) [reply tests/persist.test.js:1149]: [it didn't work ](https://github.com/ctrlplusb/easy-peasy/actions/runs/3096631134/jobs/5012402806#step:4:62)

**Review by ctrlplusb — APPROVED** [review]

**Comment by ctrlplusb** [comment]
Woot. Good stuff dude.

**Comment by jmyrland** [comment]
> Woot. Good stuff dude.

It fixes some of the tests atleast. I wasn't able to fix the `persist` tests 😞 [There was an attempt, but it didn't work.](https://github.com/ctrlplusb/easy-peasy/pull/774#discussion_r976298577)

**Pull request merged**